site/js/app.js

```diff
@@ -0,0 +1,72 @@
+
+const plLogo = `
+ ////////,,,,,,,    /////////////////////////////*                                                       
+///////###,,,,,,,,,#///////////////////////////////                                                      
+////////###(,,,,,,,,####//////////////////////////                                                       
+  ////////###,,,,,,,,,#####(/////////////////////                                                        
+   ///////////*,,,,,,,,,,,,,,,,     ////////////                                                         
+     /////////// ,,,,,,,,,,,,,,,,  ///////////                                                           
+      ///////////    ,,,,,,,,,,,,////////////                                                            
+       ,///////////    ,,,,,,,,,#//////////                                                              
+         ///////////   ,,,,,,,,##/////////                                                               
+          ////////////  ,,,,,,###///////                                                                 
+            /////////// ,,,,,,,##//////                                                                  
+             ///////////(,,,,,,,,,                                                                       
+               /////////##,,,,,,,,                                                                       
+                ////////###,,,,,,,                                                                       
+                  ///////####,,,,                                                                        
+                   ///////////                                                                           
+                    ./////////                                                                           
+                      ///////                                                                            
+--------------------------------------------------------
+Contribute at: https://github.com/PennyLaneAI/pennylane  
+--------------------------------------------------------
+`;
+
+// a key map of allowed keys
+const allowedKeys = {
+  37: 'left',
+  38: 'up',
+  39: 'right',
+  40: 'down',
+  65: 'a',
+  66: 'b'
+};
+
+// the 'official' Konami Code sequence
+const konamiCode = ['up', 'up', 'down', 'down', 'left', 'right', 'left', 'right', 'b', 'a'];
```

Comment:
We can also try add this one to the easter egg library.

Reply:
Yeah, i was thinking the same, there's actually a react package that wraps this: https://www.npmjs.com/package/react-konami-code